bitcoin-magazine/bitcoin-2024/making-bitcoin-more-private-with-cisa.md

```diff
@@ -1,71 +1,76 @@
+And joining me today on the stage, I have Jameson Lopp from CASA, Craig Raw of Sparrow Wallet and Fabien Jahr of Brink.
 So welcome them to the stage.
 Really appreciate it.
-Yeah.
 Great.
 So we're excited to be talking to you guys about this.
 I think it'd be great maybe to start off hearing a little bit more about who's on our panel today.
-So if panelists, you could tell the audience about the project that you're working on and what your first, like where you first heard about SISA from, if that makes sense.
+So if panelists, you could tell the audience about the project that you're working on and what your first, like where you first heard about CISA from, if that makes sense.
 So Craig, do you want to start?
 
-Speaker 2: 00:00:55
+Craig Raw: 00:00:55
 
 So I built Sparrow Wallet.
 It's a security and privacy focused wallet.
-The first time I heard about Caesar was really from you know other privacy activists in the Bitcoin space who were talking about you know how they were really hoping that this cross-input signature aggregation was going to be shipped as part of the taproot upgrade and Obviously, we know that that didn't happen so when you know you're looking at things from a privacy point of view you want to do things like create multi-party transactions and as we will hear, cross-input signature aggregation really provides an interesting basis for being able to do that on a more economic, with more economic benefits.
+The first time I heard about CISA was really from you know other privacy activists in the Bitcoin space who were talking about how they were really hoping that this cross-input signature aggregation was going to be shipped as part of the Taproot upgrade.
+Obviously, we know that that didn't happen so when you know you're looking at things from a privacy point of view you want to do things like create multi-party transactions and as we will hear, cross-input signature aggregation really provides an interesting basis for being able to do that with more economic benefits.
 So that was the first time that I kind of really started to look at it was from the privacy angle.
 
-Speaker 0: 00:01:54
+Fabian Jahr: 00:01:54
 
 Yeah, I primarily work on Bitcoin Core and I can't really remember a specific time when I heard about it for the first time.
-Like the time between the SecWit Soft fork and the Taproot Soft fork was really when I got very, got deeper into Bitcoin and like started contributing to Bitcoin Core and somehow it was always there.
-And I only, when I started researching on Caesar and going deeper into it over the last couple of months I saw also that yeah historically it was a topic that was brought up and that was discussed to be part of the Taproot soft fork, but then at some point it was cut basically to keep the scope smaller.
+Like the time between the SegWit Softfork and the Taproot Softfork was really when I got deeper into Bitcoin and like started contributing to Bitcoin Core and somehow it was always there.
```

Comment:
Here you write soft fork as "Softfork" and later as "softfork", but on other occasions you keep it as "soft fork" or "soft-fork". I know that we still don't point to a specific style guide but you should at least have a single way of referring to it in a transcript that you are editing.

Please keep "soft fork" as specified here: https://github.com/bitcoinops/bitcoinops.github.io/blob/master/STYLE.md